mitosheet/mitosheet/public/v3/formatting.py

```diff
@@ -0,0 +1,36 @@
+from openpyxl import Workbook
+from openpyxl.worksheet.worksheet import Worksheet
+
+from openpyxl.styles import Font, PatternFill
+from openpyxl.styles import NamedStyle
+
+def add_formatting_to_excel_sheet( params: dict[Workbook, str, dict]) -> None:
+    """
+    Adds formatting to the sheet_name, based on the formatting the user
+    currently has applied in the frontend. 
+
+    NOTE: this is a Mito Pro feature.
+    """
+    workbook = params['workbook']
+    sheet_name = params['sheet_name']
+    format = params['format']
+    sheet = workbook.get_sheet_by_name(sheet_name)
+    
+    # Add formatting to the header row    
+    header_name = f"{sheet_name}_Header"
+    if format.get('headers'):
+        header_format = NamedStyle(name=header_name)
+        # Add font and background colors to the header format
+        if format['headers']['color']:
+            font_color = format['headers']['color'][1:]
```

Comment:
Can you leave a comment explaining the 1: -- assuming this is chopping a # of the hex, but it would be great to confirm.

Reply:
Done!